spec/support/test_app_template.rb

```diff
@@ -9,21 +9,37 @@ def source_paths
   ]
 end
 
-def skip_active_job?
+def skip_active_storage?
+  true
+end
+
+def cloud_gov_organization
+  "sandbox-gsa"
+end
+
+def cloud_gov_staging_space
+  "staging"
+end
+
+def cloud_gov_production_space
+  "prod"
```

Comment:
[question (nb)] I think I remember reading some actions triggered just for `production` branch, I don't know what kind of background there is behind `prod` v `production` but I wondered if they should match.

Reply:
These are the defaults because when a new cloud.gov organization is created they come by default with `dev`, `staging`, and `prod` spaces already created, and the old terraform did not attempt to create the spaces itself, it only used what was there already. Now that we're creating the space in the terraform anyway the defaults aren't as useful so we can update if we want them to match.